docs/rule/attributes-order.md

```diff
@@ -0,0 +1,60 @@
+# attributes-order
+
+Element attributes must be ordered, by default the order is:
+
+For angle bracket invocation syntax:
+
+- arguments
+- attributes
+- element modifiers
+- ...attributes
+- ?attributes
+
+For mustache/curly invoication syntax:
```

Comment:
So there's only one type (`attributes`) for mustache/curly? No other possible types? To make sure I'm understanding this correctly, that means the only affect of this rule on mustache/curly invocation is alphabetizing the attributes? If that's the case, maybe we could say that more clearly like this:

```md
Element attributes must be ordered. By default, the order is:

* ...
* ...
* ...
* ...

Note that this ordering only applies to angle bracket invocation syntax, as mustache/curly invocation syntax only includes a single type `attributes`.
```